src/giftMockData.js

```diff
@@ -5,7 +5,7 @@ import gift3 from "./assets/gift3.png";
 export const giftMockData = {
   "data": [
     {
-      "id": "3211",
+      "id": "1",
```

Comment:
I'm not sure why I wrote the friends' id and not the gift id :sweat_smile: Thanks for catching this!